mitodl/mitxonline
# Upgrade js dependencies

#### Pre-Flight checklist



#### What are the relevant tickets?
N/A

#### What's this PR do?
Upgrades some out-of-date dependencies

#### How should this be manually tested?
App should build and run without error (logging in is probably adequate to test this).
